### PR TITLE
add KEF integration docs

### PR DESCRIPTION
**Pull request by basnijholt**
**Description:**

**Pull request in home-assistant:** home-assistant/home-assistant#28959

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/kef.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Configuration
+
+```yaml
+# Example configuration.yaml entry
+- platform: kef
+  host: IP_ADDRESS
+  type: LS50
+```
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_integrations/kef.markdown:29]: I'm missing the `port` configuration variable

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/kef.markdown (hunk 1 above)
Comment (basnijholt) [reply source/_integrations/kef.markdown:29]: Thanks @klaasnicolaas for your careful reading! I've added it in dc1d1d2.

**Comment by basnijholt** [comment]
I think this should be merged because https://github.com/home-assistant/home-assistant/pull/28959 has been merged.

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**